### ISSUE TITLE
Problems with macos 10.11.6

### ISSUE THREAD

**Issue by wassimseif** [Nightonke/Gitee#12]
Downloaded , logged in.
As soon as i try to open it from the status bar. it crashes, and disappears from the status bar. this is the stack trace from the logs

```
2017-03-17 07:27:33.784 VHGithubNotifier[37760:17506559] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[3]'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000108b5d4f2 __exceptionPreprocess + 178
	1   libobjc.A.dylib                     0x0000000107131f7e objc_exception_throw + 48
	2   CoreFoundation                      0x0000000108a577de -[__NSPlaceholderDictionary initWithObjects:forKeys:count:] + 318
	3   CoreFoundation                      0x0000000108a5766b +[NSDictionary dictionaryWithObjects:forKeys:count:] + 59
	4   VHGithubNotifier                    0x00000001051b7703 -[VHGithubNotifierManager(UserNotification) addNotifications:] + 1148
	5   VHGithubNotifier                    0x000000010519a838 __59-[VHGithubNotifierManager(Notification) remindNotificatons]_block_invoke + 249
	6   libdispatch.dylib                   0x000000010b58093d _dispatch_call_block_and_release + 12
	7   libdispatch.dylib                   0x000000010b57540b _dispatch_client_callout + 8
	8   libdispatch.dylib                   0x000000010b57929b _dispatch_root_queue_drain + 1890
	9   libdispatch.dylib                   0x000000010b578b00 _dispatch_worker_thread3 + 91
	10  libsystem_pthread.dylib             0x000000010b9354de _pthread_wqthread + 1129
	11  libsystem_pthread.dylib             0x000000010b933341 start_wqthread + 13
)
libc++abi.dylib: terminating with uncaught exception of type NSException

```

**Comment by Nightonke** [comment]
Please try release [version 1.0.2.1](https://github.com/Nightonke/Gitee/releases).

**Comment by wassimseif** [comment]
Works for a bit longer then crashes at random scenarios. When it's working there's a problem in the UI. please see screenshot attached 
![screen shot 2017-03-17 at 8 24 40 am](https://cloud.githubusercontent.com/assets/7282773/24031502/b144dd0c-0aeb-11e7-9cb8-2cf024b3451d.png)
Where should i upload the logs ?

**Comment by Nightonke** [comment]
You can send log files to 2584541288@qq.com. Thanks.

**Comment by Nightonke** [comment]
@wassimseif  Thank you so much for the log files you just sent to me! Some bugs are fixed in the newest release [1.0.2.2](https://github.com/Nightonke/Gitee/releases)

**Comment by wassimseif** [comment]
Crashing Issue fixed! still the UI problem
